### ISSUE TITLE
box_sdk_gen/internal not found with 0.6.0 pypi release artifact

### ISSUE THREAD

**Issue by akrherz** [box/box-python-sdk-gen#76]
- [x] I have checked that the [SDK documentation][sdk-docs] doesn't solve my issue.
- [x] I have checked that the [API documentation][api-docs] doesn't solve my issue.
- [x] I have searched the [Box Developer Forums][dev-forums] and my issue isn't already reported (or if it has been reported, I have attached a link to it, for reference).
- [x] I have searched [Issues in this repo][github-repo] and my issue isn't already reported.

### Description of the Issue

I am failing to build box_sdk_gen 0.6.0 due to the internal directory not being included in the source distribution.

### Steps to Reproduce

```
$ wget https://pypi.io/packages/source/b/box-sdk-gen/box-sdk-gen-0.6.0.tar.gz
$ tar -tzf box-sdk-gen-0.6.0.tar.gz  | grep internal
$
```

### Error Message, Including Stack Trace

```
'box_sdk_gen'
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/box-sdk-gen_1709797498728/test_tmp/run_test.py", line 2, in <module>
    import box_sdk_gen
  File "/home/conda/feedstock_root/build_artifacts/box-sdk-gen_1709797498728/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.12/site-packages/box_sdk_gen/__init__.py", line 1, in <module>
    from .box import *
  File "/home/conda/feedstock_root/build_artifacts/box-sdk-gen_1709797498728/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.12/site-packages/box_sdk_gen/box/__init__.py", line 3, in <module>
    from .token_storage import *
  File "/home/conda/feedstock_root/build_artifacts/box-sdk-gen_1709797498728/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.12/site-packages/box_sdk_gen/box/token_storage.py", line 5, in <module>
    from ..schemas import AccessToken
  File "/home/conda/feedstock_root/build_artifacts/box-sdk-gen_1709797498728/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.12/site-packages/box_sdk_gen/schemas.py", line 5, in <module>
    from box_sdk_gen.internal.base_object import BaseObject
ModuleNotFoundError: No module named 'box_sdk_gen.internal'
```

refs conda-forge/box-sdk-gen-feedstock#6

**Comment by mwwoda** [comment]
New package with fix was just released (0.6.1). Please try to upgrade to this version.

**Comment by akrherz** [comment]
Looks good, thank you!
